pkg/apiserver/apiserver.go

```diff
@@ -91,9 +90,8 @@ type APIGroupVersion struct {
 	// schema like api.Status, api.DeleteOptions, and api.ListOptions. Other implementors may
 	// define a version "v1beta1" but want to use the Kubernetes "v1" internal objects. If
 	// empty, defaults to Version.
-	// TODO: caesarxuchao: ServerVersion actually contains "group/version",
-	// refactor it to avoid confusion.
-	ServerVersion string
+	// TODO this seems suspicious.  Is this actually just "unversioned" now?
+	ServerGroupVersion *unversioned.GroupVersion
```

Comment:
> @liggitt, ServerGroupVersion is the versioned used by encode DeleteOptions ListOptions etc. We may not need it anymore since we are moving these things to unversioned.

I would expect to need it (or something similar) to keep v1 returning "v1" instead of "" for unversioned object types, but I haven't swept how this is used to be sure.